CMakeLists.txt

```diff
@@ -120,9 +120,82 @@ foreach(tutorial_file ${tutorial_files})
 endforeach()
 add_custom_target(move_artifacts DEPENDS ${stamp_file} ${tutorial_files_builddir})
 
+add_subdirectory (interpreter)
```

Comment:
Because ROOT_CLING_CXX_HEADERS_LOCATION is set by the interpreter and we use this to configure the modulemaps. Only solution I can think of is setting the variable here in the top-level CMakeLists. I'll make another patch for this next week.